src/core/common/message.hpp

```diff
@@ -975,6 +978,7 @@ class Message : public otMessage, public Buffer, public GetProvider<Message>
      *
      */
     bool IsChildPending(void) const;
+#endif
```

Comment:
Per our style guide, we add comment to `#endif` directives only when they are more than 20 lines away from the corresponding `#if`. 
```suggestion
#endif // OPENTHREAD_FTD
```